src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -490,36 +488,16 @@ class HttpGoogleServicesDAO(
       val list = blocking {
         executeGoogleRequest(fetcher)
       }
-      Option(list.getBillingAccounts.toSeq).getOrElse(Seq.empty)
+      // option-wrap getBillingAccounts because it returns null for an empty list
+      Option(list.getBillingAccounts).map(_.toSeq).getOrElse(Seq.empty)
     }) {
-      case e: HttpResponseException if e.getStatusCode == StatusCodes.Forbidden.intValue =>
-        getGoogleErrorMessage(e) match {
-          case Success(googlyError) if googlyError == "Request had insufficient authentication scopes." =>
-            throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.Forbidden, s"""{"requiredScopes" : [%s]}""".format(billingScopes.head) ))
-          case Success(googlyError) =>
-            throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.Forbidden, googlyError))
-          case Failure(regret) =>
-            throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.Forbidden, e.getContent))
-        }
+      case gjre: GoogleJsonResponseException
+        if gjre.getStatusCode == StatusCodes.Forbidden.intValue &&
```

Comment:
Oh I see. So where I was returning Forbidden in other Google (and not-Google) errors, you're just throwing it back up the chain. Which will be a 500. That's probably more right.